code/tyler/django/tyler_django_lab-2/blog_proj/static/css/normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
I don't think it's necessary to use `normalize.css` when bootstrap is involved. Bootstrap should have that taken care of